pdp/config.yaml

```diff
@@ -1,17 +1,15 @@
-app_root: 'http://tools.pacificclimate.org/dataportal'
-data_root: 'http://tools.pacificclimate.org/dataportal/data'
+app_root: 'https://tools.pacificclimate.org/dataportal'
+data_root: 'https://tools.pacificclimate.org/dataportal/data'
 title: ''
 ensemble_name: ''
 dsn: postgresql://httpd_meta@atlas.pcic/pcic_meta # add password for production
 pcds_dsn: postgresql://httpd@atlas.pcic/crmp # add password for production
 js_min: false
-geoserver_url: http://atlas.pcic.uvic.ca/geoserver/ # change for production
+geoserver_url: https://atlas.pcic.uvic.ca/geoserver/ # change for production
 ncwms_url: 
-    - http://atlas.pcic.uvic.ca/ncWMS/wms # change for production
+    - https://atlas.pcic.uvic.ca/ncWMS/wms # change for production
 tilecache_url: 
-    - http://a.tiles.pacificclimate.org/tilecache/tilecache.py
-    - http://b.tiles.pacificclimate.org/tilecache/tilecache.py
-    - http://c.tiles.pacificclimate.org/tilecache/tilecache.py
+    - http://tiles.pacificclimate.org/tilecache/tilecache.py
```

Comment:
Yes: SSL wildcard certs. Can't have *.*.rootdomain.org, only *.rootdomain.org. Should actually be https, but the tileserver wasn't live over https at the time.